### PR TITLE
[ONNX] Add FX exporter MaxPool tests

### PR DESCRIPTION
**Pull request by titaiwangms**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #102773

Need https://github.com/microsoft/onnxscript/pull/757

### CODE REVIEW

**Comment by BowenBao** [comment]
CI failure is "unexpected success", might want to check if it actually fixes `resnet18` under dynamic=True.

It appears a lot of xfail were still written with reason `RuntimeError: Unknown call_function target: aten.mean.dim`. Now that the op is supported, it means there are still other failures.

**Comment by titaiwangms** [comment]
> CI failure is "unexpected success", might want to check if it actually fixes `resnet18` under dynamic=True.
> 
> It appears a lot of xfail were still written with reason `RuntimeError: Unknown call_function target: aten.mean.dim`. Now that the op is supported, it means there are still other failures.

Resnet18 is supported now.

File: torch/onnx/_internal/fx/op_validation.py [hunk 1]
```diff
@@ -231,7 +231,7 @@ def _fx_args_to_torch_args(
                 )
         elif isinstance(arg, Sequence):
             wrapped_args.append(_fx_args_to_torch_args(arg))
-        elif isinstance(arg, (int, float, torch.dtype)):
```
Comment (BowenBao) [inline torch/onnx/_internal/fx/op_validation.py:234]: Could you explain why this change?

Comment (titaiwangms) [reply torch/onnx/_internal/fx/op_validation.py:234]: In aten::convolution, the bias can be passed as None in args...

Comment (titaiwangms) [reply torch/onnx/_internal/fx/op_validation.py:234]: The whole story is that resnet18 raises another issue which only fails when op_level_debug is True. And the reason is simply that the None arg is not in any condition, and go straight to the ValueError. So now, we just append it to the args as if it's normal value.

**Review by BowenBao — APPROVED** [review]
